lib/src/screens/seed/warning_page.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:cake_wallet/utils/responsive_layout_util.dart';
+import 'package:cake_wallet/themes/extensions/cake_text_theme.dart';
+import 'package:cake_wallet/view_model/advanced_privacy_settings_view_model.dart';
+import 'package:cw_core/wallet_type.dart';
+import 'package:cake_wallet/src/screens/base_page.dart';
+import 'package:cake_wallet/src/widgets/primary_button.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+import 'package:cake_wallet/routes.dart';
+import 'package:cake_wallet/themes/theme_base.dart';
+import 'package:flutter/material.dart';
+
+class WarningPage extends BasePage {
+  WarningPage(this.seedPhraseLength)
+      : isPreSeedPage = seedPhraseLength != null,
+        imageLight = Image.asset('assets/images/pre_seed_light.png'),
+        imageDark = Image.asset('assets/images/pre_seed_dark.png');
+
+  final Image imageDark;
+  final Image imageLight;
+  final int? seedPhraseLength;
+  final bool isPreSeedPage;
+
+  @override
+  Widget? leading(BuildContext context) => isPreSeedPage ? null : super.leading(context);
+
+  @override
+  String? get title => S.current.pre_seed_title;
+
+  @override
+  Widget body(BuildContext context) {
+    final image = currentTheme.type == ThemeType.dark ? imageDark : imageLight;
+
+    return WillPopScope(
+        onWillPop: () async => false,
```

Comment:
this should be `false` only in case of `isPreSeedPage` so that the user can go back normally in case of 2FA

Reply:
fixed